workers/ohsome_quality_analyst/oqt.py

```diff
@@ -169,21 +186,45 @@ async def _create_indicator(
 
 
 @create_indicator.register
-async def _create_indicator(  # noqa
-    parameters: IndicatorBpolys,
-    force: bool = False,
-) -> Indicator:
+async def _create_indicator(parameters: IndicatorBpolys, *_args) -> Indicator:  # noqa
+    """Create an indicator from scratch."""
+    name = parameters.name.value
+    layer: Layer = from_dict(
+        data_class=LayerDefinition,
+        data=get_layer_definition(parameters.layer_name.value),
+    )
+    feature = parameters.bpolys
+
+    logging.info("Calculating Indicator for custom AOI ...")
+    logging.info("Indicator name: " + name)
+    logging.info("Layer name:     " + layer.name)
+
+    indicator_class = name_to_class(class_type="indicator", name=name)
+    indicator = indicator_class(layer, feature)
+
+    logging.info("Run preprocessing")
+    await indicator.preprocess()
+    logging.info("Run calculation")
+    indicator.calculate()
+    logging.info("Run figure creation")
+    indicator.create_figure()
+
+    return indicator
+
+
+@create_indicator.register
+async def _create_indicator(parameters: IndicatorData, *_args) -> Indicator:  # noqa
```

Comment:
see above

Reply:
There is. Added exact flake8 warning code.